Library/Core/UnoCore/Targets/Android/app/build.gradle

```diff
@@ -4,22 +4,47 @@ apply plugin: 'com.android.library'
 apply plugin: 'com.android.application'
 #endif
 
+// A configuration to mark AARs to extract native libraries from
+configurations { native_implementation }
```

Comment:
Should this be guarded by `if @(Gradle.Dependency.NativeImplementation:IsRequired)`? I mean, the rest of this feature is...

Reply:
I added the guards because I wanted to avoid any overhead of the `tasks.whenTaskAdded {}`-logic, and not add non-existing search-paths when the feature is not used.

I don't know what makes most sense TBH (guards or not), maybe the guards make things a bit more explicit?

Reply:
I dunno, I have no idea if this line cause any trouble on its own... ;)

Reply:
This shouldn't cause any trouble or significant overhead.

When thinking about it... Let's just add the last guard to make it explicit for the reader that these chunks of code are related. Hold on! :)